packages/auth/test/integration/flows/anonymous.test.ts

```diff
@@ -65,17 +66,19 @@ describe('Integration test: anonymous auth', () => {
 
   context('email/password interaction', () => {
     let email: string;
+    let password: string;
 
-    beforeEach(() => {
+    beforeEach(async () => {
       email = randomEmail();
+      password = await generateValidPassword(auth);
```

Comment:
hmm.. this means if the test is run against the real backend(instead of emulator), we will use "aA0.." as the password?

I guess that might be ok, but can we just define a "getBasicPassword()" function that returns "password" or just use a const?

Reply:
I can change them to use a const (which is essentially what was here before), but I had updated this so it wouldn't fail due to the password not meeting the requirements if these tests were run with a project that had a policy configured.

Reply:
Based on our conversation offline, I'll remove these calls and replace them with a comment mentioning that generateValidPassword can be used to get a password that complies with the policy if they choose to run these tests with their own project.